package.json

```diff
@@ -16,7 +16,8 @@
         "lint": "eslint --ext .js,.vue src test/unit test/e2e/specs",
         "build": "node build/build.js",
         "electron": "NODE_ENV=DEV electron electron.js",
-        "test-api": "babel-node --presets es2015 test_api.js"
+        "test-api": "babel-node --presets es2015 test_api.js",
+        "postbuild": "rm -rf app/ && mkdir app app/templates && cp dist/index.html app/templates && cp -a dist/static app/ && cp app.py app/"
```

Comment:
Mix up server and client code here is NG.
`package.json` only in charge of js parts.

Reply:
this way is convenient

Reply:
May move it to other file instead of in `package.json`.
If you have to do so, you may use nodejs express instead of python flask.
File operations on python files within `package.json` is really not good, because python is the server part and package.json is only for the client part. People used to look up the "post build" of python server in a python config file.

Reply:
can nodejs express be deployed using zappa?

Reply:
nodejs code can be deployed by npm pkg `serverless` (the same as zappa for python)

Reply:
why not add `serverless` to this hot wallet so that this can be easily deployed?

Reply:
- `serverless` is should be installed in environment globally (just as zappa). The content of pj does not depend on it.
- This hot wallet is not including any server. This patch adding a python server is just to make things easy to deploy the code onto lambda. It is just the same thing to adding a nodejs server.

Reply:
so npm is not a server?

Reply:
npm is just like pip....

Reply:
I thought it was like a server because inside README there is 
```
 npm run dev
```
this look like it can do same thing like flask

Reply:
Yes, it's a mock server only for development. Not for production.

Reply:
Maybe you should add `serverless` or 'flask' thing to Dockerfile and see what we can do from there?

Reply:
- For deployment onto an server instance nginx is just fine, 
- For serverless deployment, setting cloudFront could be a better practice in my opinion. Because adding an extra server may lead to twice data traffic (S3 -> lambda(static website files + python server package file), lambda -> ApiGateway(website files)).
- However if you mean to use an extra server in serverless deployment, I suggest that "nodejs+express+serverless" could be a better choice than "python+flask+zappa" according to the tech stack of the repo, which things to do is just 'translate' your python code to nodejs one.